Gemfile

```diff
@@ -1,118 +1,111 @@
 gem 'omniauth'
-gem 'omniauth-oauth2', '~> 1.3.1'
 gem 'omniauth-github', '~> 1.1.2'
 gem 'omniauth-gplus', '~> 2.0.1'
-gem 'font-awesome-rails'
-gem 'acts_as_tree'
-gem 'acts_as_follower'
-gem 'will_paginate-bootstrap'
-gem 'coveralls', '~> 0.7.9', require: false # measure test coverage
-gem 'friendly_id'  # for more REST-ful routes, use human-readable IDs
-gem 'colored' # colorizing console
-gem 'redcarpet' # renders markdown
-gem 'coderay' # syntax highlighting for markdown code blocks
-gem 'acts-as-taggable-on' # Add tags to objects. Used on Projects
-gem 'geocoder' # geocoding
-gem 'paper_trail', '4.0.0' # version control for Document
-gem 'verbs'   # language and verbs - not used for now but I plan to use it in Events /Thomas
-gem 'ice_cube', '0.11.1'     # used to generate event schedules, locked to last known version not to have memory leaks
-gem 'jquery-turbolinks', '2.0.2'    #fix for turbolink problem we had with the HOA button and jQuery not loading ??
-gem 'addressable'       # used for uri validation
-gem 'pivotal-tracker-api', '0.1.9' # used for Pivotal Tracker API v5
-gem 'exception_notification'
-gem 'acts_as_votable', '~> 0.10.0' #Allows WSO to track member's votes on votable objects (articles, comments, ...)
-gem 'utf8-cleaner'
-gem 'public_activity' #Create activity feed
-gem 'nokogiri', '~> 1.6.7.2'
-gem 'rails_autolink', '~>1.1.6'
-
-gem 'yui-compressor'
-gem 'compass-rails'
+gem 'omniauth-oauth2', '~> 1.3.1'
+gem 'paper_trail', '4.0.0'
+gem 'pg'
+gem 'pivotal-tracker-api', '0.1.9'
+gem 'polyamorous', '~> 1.2.0'
+gem 'public_activity'
+gem 'puma'
 gem 'rack-cache'
-#gem 'sprockets-image_compressor', '~> 0.2.4'
-#gem 'sprockets-webp'
+gem 'rails', '4.2.6'
+gem 'rails_autolink', '~>1.1.6'
+gem 'redcarpet'
+gem 'sass-rails'
 gem 'sprockets-rails', '2.3.3'
-gem 'eventmachine', '~> 1.0.7'
-
-gem 'local_time', '~> 1.0.3'
-gem 'config' # a gem to manage configuration files
-gem 'nearest_time_zone' # find the name of a timezone for a latitude and longitude without relying on a web service
-gem 'octokit' # Ruby wrapper for the GitHub API
-gem 'sucker_punch' # Single Process Ruby asynchronous processing library
-gem 'twitter', '~> 5.11.0' # twitter api wrapper
-gem 'jvectormap-rails', '~> 1.0.0' #jVectorMap for the Rails asset pipeline
-
-gem 'active_model-errors_details'
-
 gem 'stripe'
-gem 'kaminari'
-
+gem 'sucker_punch'
+gem 'therubyracer', platforms: :ruby
+gem 'turbolinks'
+gem 'twitter', '~> 5.11.0'
+gem 'uglifier'
+gem 'utf8-cleaner'
+gem 'verbs'
+gem 'will_paginate-bootstrap'
+gem 'yui-compressor'
 
 group :test do
-  gem 'capybara' # Simulates user actions for cucumber
-  gem 'cucumber-rails', :require => false # Cucmber features
-  gem 'capybara-webkit'  # Headless driver for capybara
-  gem 'selenium-webdriver' # Headful driver for capybara
-  gem 'poltergeist'  # yet another headless driver for capybara
+  gem 'capybara'
+  gem 'capybara-webkit'
+  gem 'capybara-screenshot'
+  gem 'cucumber-rails', :require => false
```

Comment:
I read about that in the past and forgot, thanks for remind me. This will make our gemfile more readable and clean.